sympy/physics/tests/test_secondquant.py

```diff
@@ -1219,3 +1219,17 @@ def test_internal_external_pqrs_AT():
     ]
     for permut in exprs[1:]:
         assert substitute_dummies(exprs[0]) == substitute_dummies(permut)
+
+
+def test_canonical_ordering_AntiSymmetricTensor():
+    v = symbols("v")
+    virtual_indices = ('c', 'd')
+    occupied_indices = ('k', 'l')
+
+    c, d = symbols(('c','d'), above_fermi=True,
+                                   cls=Dummy)
+    k, l = symbols(('k','l'), below_fermi=True,
+                                   cls=Dummy)
+
+    print(AntiSymmetricTensor(v, (k, l), (d, c)))
```

Comment:
doctests will check what is printed, but tests require an assertion to be made. In this case I believe the canonical result will pass `assert AntiSymmetricTensor(v, (k, l), (d, c)) == -AntiSymmetricTensor(v, (l, k), (d, c))`...but check to see that this matches the output that was given by your print statement. Summary: replace your print statement with the assert statement and remove the comment (which demonstrated that the issue was valid).